jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -465,6 +465,7 @@
   (:pick-object-with-movable-region
     (arm movable-region &key (n-trial 1) (n-trial-same-pos 1)
          (do-stop-grasp nil) (grasp-style :suction))
+    "Return value: :grasp-succeed, :grasp-fail or :ik-fail"
```

Comment:
`:grasp-success` `:grasp-failure` and `:ik-failure`
or
`:grasp-succeeded` `:grasp-failed` and `:ik-failed`